src/vm_cpu.rs

```diff
@@ -12,6 +12,9 @@ use riscv_page_tables::GuestStagePagingMode;
 use riscv_pages::{
     GuestPhysAddr, GuestVirtAddr, InternalClean, PageOwnerId, RawAddr, SequentialPages,
 };
+#[cfg(target_feature = "v")]
+use riscv_regs::VectorRegister;
```

Comment:
Since this `use` is only used once, you can just put it in `guest_vpr_offset()` and avoid the extra `#cfg` directive.

Reply:
will fix